context/app/static/js/components/Header/UserLinks/UserLinks.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Dropdown from '../Dropdown';
+import DropdownLink from '../DropdownLink';
+import { StyledDivider } from '../HeaderContent/style';
+import { TruncatedSpan, WarningSpan } from './style';
+
+function UserLinks(props) {
+  const { isAuthenticated, userEmail } = props;
+
+  const displayName = userEmail || 'User';
```

Comment:
Since this is only used once, you could move it inline, but if you think having a variable name adds readability that works as well.